Dockerfile

```diff
@@ -54,24 +46,31 @@ RUN ./run_tests.sh "${PYTEST_FILES}"
 # Bastion container spec.                                                      #
 ################################################################################
 
+# Bastion container for application deployment.
```

Comment:
nit: These comments are already covered in the comment blocks for each build target, e.g. L46 or L59.